painttiming.bs

```diff
@@ -62,9 +83,25 @@ Formally, we consider the user agent to have "rendered" a document when it has p
 
     NOTE: The rendering pipeline is very complex, and the timestamp should be the latest timestamp the user agent is able to note in this pipeline (best effort). Typically the time at which the frame is submitted to the OS for display is recommended for this API.
 
+<dfn>Contentful</dfn>: An element counts as contentful when one or more of the following apply:
+* The element has a [=text node=] descendant, with at least one character, excluding [=document white space characters=].
+* The element is a [=replaced image=] representing a [=loaded=] image.
+* The element's current style contains a [=loaded=] [=background-image=] or [=mask-image=].
```

Comment:
"current style" isn't a precise term. It should be "used" style I think.

Inclusion of [=mask-image=] is surprising. Masks just hide things, they don't make things visible.

Reply:
An image mask on a solid color background would make it look like the image. But I think it's ok if currently mask-images would not count as contentful.